rules.bzl

```diff
@@ -16,6 +16,15 @@ def _buildbuddy_toolchain_impl(rctx):
     else:
         default_platform = "platform_linux"
 
+    default_container_image = _container_image_prop(rctx.attr.container_image)
```

Comment:
nit: rename this to `container_image`, since you're reading the `container_image` attribute passed by the user, not the default?

Reply:
ehh nevermind I think the previous code already had this named `default_container_image`